### PR TITLE
Add blog post to announce python-tuf 1.0.0 release

### PR DESCRIPTION
**Pull request by lukpueh**

### CODE REVIEW

**Comment by lukpueh** [comment]
cc @jku

File: _posts/2022-02-21-tuf-1_0_0.md [hunk 1]
```diff
@@ -0,0 +1,55 @@
...
+  A low-level interface for both consuming and creating TUF metadata. Metadata
+  API is a flexible and easy-to-use building block for any higher level tool or
+  library.
+
+Python-TUF 1.0.0 is the result of a comprehensive rewrite of the project,
+removing several hard to maintain modules and replacing them with safer and
+easier to use APIs:
+- The project was reduced from 4700 lines of hard to maintain code to 1400
+  lines of modern, maintainable code
+- The implementation details are now easier to reason about which should
```
Comment (jhdalek55) [inline _posts/2022-02-21-tuf-1_0_0.md:41]: Add comma after "reason about"

File: _posts/2022-02-21-tuf-1_0_0.md [hunk 2]
```diff
@@ -0,0 +1,55 @@
...
+  accelerate future improvements on the project
+- Metadata API provides a solid base to build other tools on top of – as proven
+  by the ngclient implementation and the [repository code
+  examples](https://github.com/theupdateframework/python-tuf/tree/develop/examples/repo_example)
+- Both new APIs are highly extensible and allow application developers to
+  include custom network stacks, file storage systems or public-key
+  cryptography algorithms, while providing easy-to-use default implementations
+
+With this foundation laid, Python-TUF developers are currently planning next
+steps. You can expect at least improved repository side tooling but we’re open
```
Comment (jhdalek55) [inline _posts/2022-02-21-tuf-1_0_0.md:51]: At the very least, you can expect improved repository side tooling, but we're also open to new ideas. Pop in to

**Review by jhdalek55 — CHANGES_REQUESTED** [review]

File: _posts/2022-02-21-tuf-1_0_0.md [hunk 3]
```diff
@@ -0,0 +1,55 @@
...
+Framework](https://theupdateframework.io/) specification, an open source
+framework for securing content delivery and updates. It protects against
+various types of supply chain attacks and provides resilience to compromise.
+
+For the past 7 releases the project has introduced new designs and
+implementations, which have gradually formed two new stable APIs:
+- [`ngclient`](https://theupdateframework.readthedocs.io/en/latest/api/tuf.ngclient.html):
+  A client API that offers a robust internal design providing implementation
+  safety and flexibility to application developers.
+- [`Metadata API`](https://theupdateframework.readthedocs.io/en/v0.20.0/api/tuf.api.html):
```
Comment (jku) [inline _posts/2022-02-21-tuf-1_0_0.md:31]:
```suggestion
- [`Metadata API`](https://theupdateframework.readthedocs.io/en/latest/api/tuf.api.html):
```
I fixed this in the google doc already

**Comment by jhdalek55** [comment]
LGTM--I think its ready to post

**Review by JustinCappos — APPROVED** [review]

**Pull request merged**